concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1136,24 +1136,24 @@ Hale-1973-1798-1134	Human_Mind_and_Conditions-22	pain	n	10a.22	1783	PAIN
 Hale-1973-1798-1135	Human_Mind_and_Conditions-23	weeping	n	10a.23		
 Hale-1973-1798-1136	Human_Mind_and_Conditions-24	weakness	n	10a.24		
 Hale-1973-1798-1137	Human_Mind_and_Conditions-25	fever	n	10a.25	1534	FEVER
-Hale-1973-1798-1138	Human_Mind_and_Conditions-26	cough	n	10a.26		
-Hale-1973-1798-1139	Human_Mind_and_Conditions-27	sneeze	n	10a.27		
-Hale-1973-1798-1140	Human_Mind_and_Conditions-28	hiccup	n	10a.28		
+Hale-1973-1798-1138	Human_Mind_and_Conditions-26	cough	n	10a.26	3913	COUGH (ACT OF COUGHING)
+Hale-1973-1798-1139	Human_Mind_and_Conditions-27	sneeze	n	10a.27	1621	SNEEZE
+Hale-1973-1798-1140	Human_Mind_and_Conditions-28	hiccup	n	10a.28	162	HICCOUGH
```

Comment:
Again, concepticon lists `162 HICCOUGH` as a verb, and since in this wordlist this is a noun, I think it should technically be unmapped.